tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -56,6 +58,19 @@ def test_add_node(self):
                 node_names=helpers.get_worker_nodes(),
                 status=constants.NODE_READY
             )
+            all_nodes = helpers.get_worker_nodes()
+
+            new_spun_nodes = list(
+                set(all_nodes) - set(initial_nodes)
+            )
+            logging.info('labeling:')
+            node_obj = ocp.OCP(kind='node')
+            for new_node in new_spun_nodes:
+                node_obj.add_label(
```

Comment:
Why can't we use this function t add node and to label the node https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/node.py#L242 ?
This function needs update to get the label as param from user.

Reply:
got it , will use add_new_node_and_label_it()